chapters/0-introduction.tex

```diff
@@ -0,0 +1,266 @@
+This book is intended to teach all users of development data
+how to handle data effectively, efficiently, and ethically.
+It takes lessons, tools, and processes that emerged from the DIME portfolio
+and compiles them into a single narrative about doing data work
+that we hope will provide a foundation for these professional skills.
+
+DIME generates high-quality and operationally relevant data and research
+to transform development policy, help reduce extreme poverty, and secure shared prosperity.
+It develops customized data and evidence ecosystems to produce actionable information
+and recommend specific policy pathways to maximize impact.
+DIME conducts research in 60 countries with 200 agencies, leveraging a
+US\$180 million research budget to shape the design and implementation of
+US\$18 billion in development finance.
+DIME also provides advisory services to 30 multilateral and bilateral development agencies.
+The DIME team includes four primary topic pillars,
+has included dozens of research economists,
+and, over the years, has employed hundreds of full-time research assistants, field coordinators, and staff.
+This group taken together, has conducted over 325 impact evaluations.
+This book exists to take advantage of that concentration and scale of research,
+to synthesize many resources for data collection and research,
+and to make DIME tools available to the larger community of development researchers.
+
+As part of its broader mission mission, DIME invests in public goods (such as this book)
+to improve the quality and reproducibility of development research around the world.
+One key early innovation at DIME was the creation of the DIME Analytics team,
+who are responsible for writing and maintaining this book.
+This team has five full-time staff and several part-time members,
+and are collectively responsible for ensuring the quality of research practices across DIME.
+This is done through an intensive, collaborative innovation cycle:
+DIME Analytics onboards and supports research assistants and field coordinators,
+provides standard tools and workflows to all teams,
+delivers intensive support when new tasks or challenges arise,
+and then develops and integrates lessons from those engagements to bring to the full team.
+In that way the Analytics team is DIME's method of ``institutionalizing''
+a set of tools and practices, developed and refined over time,
+that give the unit a common base of knowledge and practice.
+This book is targeted to everyone who interacts with development data:
+graduate students, research assistants, policymakers, and empirical researchers.
+It aims to be a highly practical resource so the reader can
+immediately begin to collaborate effectively on large, long-term research projects
+that already use the methods and tools outlined in this book.
+
+\end{fullwidth}
+
+%------------------------------------------------
+
+\section{Doing credible research at scale}
```

Comment:
this section header no longer seems applicable, at least to me it made reference to DIME's scale and how Analytics grew out of that